WordPress/Classes/Services/AppUpdate/AppUpdateCoordinator.swift

```diff
@@ -162,12 +172,23 @@ private enum Constants {
     static let lastFetchedAppStoreInfoThresholdInDays = 1
 }
 
-private extension String {
-    func isLower(than anotherVersionString: String) -> Bool {
-        self.compare(anotherVersionString, options: .numeric) == .orderedAscending
-    }
-
-    func isLowerThanOrEqual(to anotherVersionString: String) -> Bool {
-        [ComparisonResult.orderedSame, .orderedAscending].contains(self.compare(anotherVersionString, options: .numeric))
+ extension Version {
+    init?(from versionString: String) {
```

Comment:
(Nit) I would put this directly in the `Version` file`

Reply:
Fixed! [d97c8d5](https://github.com/wordpress-mobile/WordPress-iOS/pull/23224/commits/d97c8d588727ac648a393b440395585866475158)